app/assets/stylesheets/common/base/onebox.scss

```diff
@@ -640,6 +641,7 @@ aside.onebox.stackexchange .onebox-body {
 }
 
 .onebox {
+  &.allowlistedgeneric,
   &.whitelistedgeneric,
```

Comment:
Ditto https://github.com/discourse/discourse/pull/10209/files#r459248952

Reply:
As above, i think we do need to keep this